prog/weave-kube/weave-daemonset-k8s-1.6.yaml

```diff
@@ -102,7 +102,8 @@ spec:
           type: spc_t
       volumes:
         - name: weavedb
-          emptyDir: {}
+          hostPath:
+            path: /var/lib/weave-net
```

Comment:
I'd say we should unify to `/var/lib/weave/net`
**if** we foresee other open source projects will need store things here too,
**else** just `/var/lib/weave`